### PR TITLE
Azure Machine Learning WebServicesRP new API version

### PR DESCRIPTION
**Pull request by NonStatic2014**
## Description
Azure Machine Learning WebServicesRP new API version

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md).**
- [ ] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md).**
- [ ] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.
- [X] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [X] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d#what-is-the-change).
- [ ] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [X] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [X] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [X] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

File: src/ResourceManager/MachineLearning/Commands.MachineLearning/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.13.0")]
-[assembly: AssemblyFileVersion("0.13.0")]
+[assembly: AssemblyVersion("1.0.0")]
+[assembly: AssemblyFileVersion("1.0.0")]
```
Comment (cormacpayne) [inline src/ResourceManager/MachineLearning/Commands.MachineLearning/Properties/AssemblyInfo.cs]: @NonStatic2014 revert these changes as we will update these versions during the release process

File: src/ResourceManager/MachineLearning/Commands.MachineLearning/Microsoft.Azure.Commands.MachineLearning.dll-Help.psd1 [hunk 2]
```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.9.0' 
+ModuleVersion = '1.0.0'
```
Comment (cormacpayne) [inline src/ResourceManager/MachineLearning/Commands.MachineLearning/Microsoft.Azure.Commands.MachineLearning.dll-Help.psd1]: @NonStatic2014 revert these changes as we will bump the version during the release process

File: .gitignore [hunk 3]
```diff
@@ -203,3 +203,4 @@ FakesAssemblies/
 /tools/*.dll
 *.GhostDoc.xml
 pingme.txt
+tools/LocalFeed/
```
Comment (NonStatic2014) [inline .gitignore]: Oops, temporary local change. should not be included. Will remove it.

File: src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/ScenarioTests/Common.ps1 [hunk 4]
```diff
@@ -54,21 +54,6 @@ Gets the default location for a provider
 #>
 function Get-ProviderLocation($providerNamespace, $resourceType)
 {
-    if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne `
```
Comment (NonStatic2014) [inline src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/ScenarioTests/Common.ps1]: Trying to make the test execution predictable and not impacted by other cmdlet which not related to the test target.

**Comment by cormacpayne** [comment]
@NonStatic2014 Hey Klein, would you mind pulling the latest changes from the `preview` branch? These changes removed the dll-Help.xml files from the repository, so you will no longer need to update this file in your PR.

**Comment by cormacpayne** [comment]
@NonStatic2014 it looks like you are bringing in changes from older pull requests. You will need to rebase. You can do so by following the steps in this document: https://github.com/Azure/azure-powershell/blob/preview/documentation/cleaning-up-commits.md#rebasing
